base-template/contributors-guide.md

```diff
@@ -1,51 +1,250 @@
+   In the banner at the top of the page, click `Compare and Create Pull Request`.
+1. Give your PR a title, and write a description of the changes.
+   Link any relevant issues, and tag people you want to review your changes.
+   If you intend to add more commits to your PR, mark your PR as a draft so that reviewers know you have not finished.
 
 ## Choosing something to work on
 
-Compellingly iterate premium testing procedures through effective web services. Collaboratively recaptiualize client-based leadership skills rather than adaptive initiatives. Distinctively synthesize customer directed markets after worldwide portals. Appropriately exploit revolutionary e-services without excellent content. Assertively unleash clicks-and-mortar leadership skills with 2.0 action items.
+This section discusses how to find something to work on within the Good Docs Project.
 
 ### Issues
 
-Assertively enhance functional solutions for synergistic opportunities. Proactively expedite cross-media synergy whereas cutting-edge services. Appropriately embrace principle-centered leadership via leveraged infomediaries. Competently negotiate top-line products through 24/365 experiences. Professionally matrix goal-oriented processes and bleeding-edge experiences.
+Like most open source projects, there are always tasks to do.
+For the list of issues, see [https://github.com/thegooddocsproject/templates/issues](The Good Docs Project template issues).
+Look for issues with a `good first issue` tag if you want something small to start on.
 
-### Pull Requests
+When you have found an issue you want to work on, either assign the issue to yourself, or leave a comment stating that you are going to work on it before you get started.
+Add comments in the issue to ask questions or to clarify requirements if you are uncertain.
+When you have a pull request ready, link the issue in the pull request details, so that the issue closes when the PR merges.
 
-Phosfluorescently actualize economically sound alignments rather than quality synergy. Seamlessly negotiate value-added intellectual capital before equity invested applications. Intrinsicly re-engineer real-time total linkage and bleeding-edge total linkage. Monotonectally procrastinate multimedia based internal or "organic" sources vis-a-vis impactful outsourcing. Appropriately visualize cross-media quality vectors and backward-compatible vortals.
+### Reviewing pull requests
 
-### Moving out of the incubator
+The entire community reviews pull requests from others.
+The more reviews each pull request gets, the better the quality of our code.
+
+Note that pull requests in different repositories within the Good Docs Project require different levels of review rigor.
+For example, the `incubator` repo has a much lower standard of code quality than the `templates` repo.
+For a full explanation of our review standards, see [insert.link.here].
```

Comment:
> see [insert.link.here].

Specific page in mind?  Or this a followup?

Reply:
I think there was work being done on review standards in a gdoc, but I'm not sure it has a home on the site yet.